compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -5078,3 +5079,76 @@ declare_lint! {
     };
     crate_level_only
 }
+
+declare_lint! {
+    /// The `abi_error_disabled_vector_type` lint detects function definitions and calls
```

Comment:
Lint names should always be plural.

What about `abi_unsupported_vector_types`?